breaking: update documentation by setting videoCompression: false (#5301)
cypress-io/cypress-documentation: docs/guides/guides/screenshots-and-videos.mdx

}
```

> :::cypress-config-example

Author: this was missing from the video configuration example which broke the others which I missed in the previous PR

[`videosFolder`](/guides/references/configuration#Videos) which is set to
`cypress/videos` by default.

> After `cypress run` completes, videos are not compressed by default. However,

Author: @emilyrohrbough should we just remove it?

> After `cypress run` completes, videos are not compressed by default. However,

Author: I'm currently working on revamping the encoding section

`cypress/videos` by default.

After `cypress run` completes, Cypress compresses the video in order to save on
file size. By default videos do not compress, but this is configurable with the

> [`videoCompression`](/guides/references/configuration#Videos) property.

Reviewer: I think we should reword this more since starting with "After `cypress run` completes, Cypress compresses the video..." doesn't really make sense any more.

Suggestion: "After `cypress run` completes, videos are not compressed by default. However, compression can be enabled and configured with the [`videoCompression`](/guides/references/configuration#Videos) property in order to save on file size."

Author: OK I gave this a reword in f94affcc7

### Video encoding

After a video is recorded, Cypress encodes the video to a commonly digestable
format. Part of this encoding process includes video compression.

Compression is disabled by default, meaning this step will be skipped
completely, so the file size of the video will be larger, but the encoding

> process should be faster. Setting

Reviewer 2: ```suggestion
process is faster. Setting
```

Author: addressed in 718e748d0

[`videoCompression`](/guides/references/configuration#Videos) to `true` will

> coerce the value to 32 CRF, which takes longer to process, but results in a

Reviewer 2: ```suggestion
coerce the video compression value to 32 Constant Rate Factor (CRF), which takes longer to process, but results in a
```

smaller video.

**Enabling compression**

:::cypress-config-example

```ts
{
  videoCompression: true
}
```

:::

If your spec files have a long run duration and
[`videoCompression`](/guides/references/configuration#Videos) is enabled, you
might notice a time gap between a finished spec and a new spec starting during
`cypress run`. During this time, Cypress is encoding the captured video and
possibly uploading it to Cypress Cloud.

> If the machine is encoding the video slowly (which is often the case for virtual
> machines that use less CPU cores), the encoding might take a long time. In this

Reviewer 2: messy suggestion but your current paragraph hangs. /  doesn't list common scenarios:
```suggestion
**Change compression value from 32**

In addition to enabling or disabling video compress, you can specify the CRF value used to compress the video. Here is some common scenario:

- If the machine is encoding the video slowly (which is often the case for virtual
machines that use less CPU cores), increase(??) the CRF value to .....

- If your videos are extremely low quality, try decreasing the CRF value....

```

Author: gave this a shot in 206a57c0a

case, you can modify the
[`videoCompression`](/guides/references/configuration#Videos) configuration to
make the encoding a little bit faster. Here are some common scenarios:

> **Change compression value**

Reviewer 2: ```suggestion
**Change compression value from 32**
```

A lower `videoCompression` value will spend less time compressing and result in
a bigger video file size.

**Enabling compression**

:::cypress-config-example

```ts
{
  videoCompression: true
}
```

:::

Compression is disabled by default, meaning this step will be skipped

> completely, so the video will be large, but the processing should be faster.

Reviewer: ```suggestion
completely, so the file size of the video will be larger, but the processing should be faster.
```
